lib/Rex/Virtualization/Lxc.pm

```diff
@@ -0,0 +1,53 @@
+#
+# (c) Jan Gehring <jan.gehring@gmail.com>
+#
+# vim: set ts=2 sw=2 tw=0:
+# vim: set expandtab:
+
+=head1 NAME
+
+Rex::Virtualization::Lxc - Linux Containers Virtualization Module
+
+=head1 DESCRIPTION
+
+With this module you can manage Linux Containers.
+
+=head1 SYNOPSIS
+
+ use Rex::Commands::Virtualization;
+
+ set virtualization => "Lxc";
+
+ use Data::Dumper;
+
+ print Dumper vm list => "all";
+ print Dumper vm list => "active",
+   fancy => 1,
+   format => 'name,ram';
+
+ print Dumper vm list => "all",
+ fancy => 1;
+
+=cut
+
+package Rex::Virtualization::Lxc;
+
+use strict;
+use warnings;
+
+our $VERSION = '1.4.0'; # VERSION
```

Comment:
@litwol you don't need to write `our $VERSION = ...` there. Just add the comment `# VERSION`
The release process then will add the right version there.